.gitmodules

```diff
@@ -1,8 +1,8 @@
 [submodule "xpring-common-js"]
 	path = xpring-common-js
-	url = git@github.com:xpring-eng/xpring-common-js
+	url = https://github.com/dangell7/xpring-common-js.git
```

Comment:
can you revert this to point at xpring-eng?  Agree with changing to `https://` though.

Reply:
Done.